nncf/ptq/examples/yolo_v5/yolo_v5_quantization.py

```diff
@@ -0,0 +1,165 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os
+import subprocess
+from pathlib import Path
+from typing import Tuple
+
+import openvino
+from openvino.runtime import Core
+import torch
+
+from nncf import ptq
+from nncf.common.utils.logger import logger as nncf_logger
+
+from yolov5.utils.general import check_dataset
+from yolov5.utils.general import download
+from yolov5.utils.dataloaders import create_dataloader
+from yolov5.val import run as validate
+
+
+FILE = Path(__file__).resolve()
+# Relative path to the `yolo_v5` directory.
+ROOT = Path(os.path.relpath(FILE.parent, Path.cwd()))
+# Path to the directory where the original and quantized IR will be saved.
+MODEL_DIR = ROOT.joinpath('yolov5m_quantization')
+# Path to the dataset config from the `ultralytics/yolov5` repository.
+DATASET_CONFIG = ROOT.joinpath('yolov5', 'data', 'coco.yaml')
+
+
+def run_example():
+    """
+    Runs the YOLOv5 quantization example.
+    """
+    # Step 1: Prepare OpenVINO model.
+    ir_model_xml, ir_model_bin = torch_to_openvino_model()
```

Comment:
I would load the PT model in this function and use `convert_torch_to_openvino(model)`?